app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,53 @@
     <string name="help_title">Help</string>
     <string name="help_navigation_title">Navigation</string>
     <string name="help_navigation_text">
-        From the home screen, swipe down to display the favorites panel and swipe up to hide it.
+        Swipe down from the home screen to show favorites, and upwards to hide them.
         \n
-        \nIf you swipe up when the favorites panel is hidden, it will display the complete list of applications.
-        Swipe down from the top of this list to come back to the home screen.
+        \nSwipe up to show all apps if the favorites panel is hidden.
+        Swipe down from the top of it to return to the home screen.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nThe button below the favorites panel opens the launcher menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
+        Shows a popup of favorites above other apps when you leave the home screen.
         \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
+        \If instead it brings you back to the home screen:
+        \n- Press the \"clear all\" button in \"Recent Apps\" or
+        \n- Reboot your device
         \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
+        \nUnintrusively, the notification icon is transparent and of low priority.

```

Comment:
```suggestion
        \nThe transparent notification icon has low priority to avoid empty space up top.

```

Reply:
```suggestion
        \nThe transparent notification icon has low priority to avoid showing empty space at the start of the status bar.
```
I like the new proposal, but maybe "start" (or "beginning") would be more accurate than "top".